data.py

```diff
@@ -87,7 +92,76 @@ def populate(oxigraph_url=None, write_to_db=True):
         editions_source_dict = json.load(f)
 
     # Validate then add to graph
-    # TODO - add schema validation
+    class ContactPoint(BaseModel):
```

Comment:
@Moasib-Arif - not quite what I meant, I meant _use_ the `Versions` and `Editions` schemas (as imported via `from src import schemas`. See this example of using the `Datasets` schema: https://github.com/GSS-Cogs/idpd-api-poc/blob/89e99d7f0d3235bcb787b69cadc71229ac0eb7c3/data.py#L80